test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go

```diff
@@ -546,27 +536,55 @@ func setup(ctx context.Context) func(ctx2 context.Context) {
 			By(fmt.Sprintf("skipping teardown - no changes to profile %q were applied", profile.Name))
 		}
 	}
-	testprofiles.UpdateWithRetry(profile)
-	mcp, err := mcps.GetByProfile(profile)
-	Expect(err).ToNot(HaveOccurred())
-
-	mcps.WaitForCondition(mcp, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
-	mcps.WaitForCondition(mcp, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+	profiles.UpdateWithRetry(profile)
+	poolName := getPoolName()
+	CheckProfileUpdating(poolName, profile)
```

Comment:
Please use `profilesupdate.WaitForTuningUpdating`

Reply:
I did eventually, inside the function - [line](https://github.com/openshift/cluster-node-tuning-operator/blob/7ad57a846150cc2c97cf9458b6204a9611cfbaf4/test/e2e/performanceprofile/functests/11_mixedcpus/mixedcpus.go#L560) since it was used couple of times

Reply:
`profilesupdate.WaitForTuningUpdating` already checks for both HyperShift and OpenShift

Reply:
Updated with latest commit